test/prob/inv_chi_square/inv_chi_square_ccdf_log_test.hpp

```diff
@@ -56,6 +56,6 @@ class AgradCcdfLogInvChiSquare : public AgradCcdfLogTest {
                                                     const T5&) {
     using stan::math::gamma_p;
 
-    return log(1.0 - gamma_p(0.5 * nu, 0.5 / y));
+    return log(gamma_p(0.5 * nu, 0.5 / y));
```

Comment:
Here's the math version: https://github.com/stan-dev/math/blob/develop/stan/math/prim/prob/inv_chi_square_lccdf.hpp#L92

(this is checked with finite differences so I assume it's the correct one)